static/js/Header.jsx

```diff
@@ -50,7 +50,7 @@ class Header extends Component {
           { Sefaria._siteSettings.TORAH_SPECIFIC ?
           <a className="home" href="/" >{logo}</a> : null }
           <a href="/texts" className="textLink"><InterfaceText context="Header">Texts</InterfaceText></a>
-          <a href="/topics" className="textLink"><InterfaceText>Topics</InterfaceText></a>
+          <a href="/django_topics" className="textLink"><InterfaceText>Topics</InterfaceText></a>
```

Comment:
I think this (and all the diffs below changing `/topics` to `/django_topics`) might be a mistake... It doesn't seem like we set any route for this URL.

Maybe an errant find/replace?